From 96a03d5a0befdb970c1173a894f30eb81c19d02d Mon Sep 17 00:00:00 2001
From: Xiaowen Liu <b37945@freescale.com>
Date: Tue, 9 Apr 2013 10:49:02 +0800
Subject: [PATCH 446/486] ENGR00252203 fix QQ-HD video chat show colorful
 grids.

the root cause is the apk set the fixed fps to 10.
but, it is not in current fps range.
so, enlarge the fps range to include 10fps.

Signed-off-by: Xiaowen Liu <b37945@freescale.com>
---
 mx6/libcamera2/Ov5640Csi.cpp  | 4 ++--
 mx6/libcamera2/Ov5640Mipi.cpp | 4 ++--
 mx6/libcamera2/Ov5642Csi.cpp  | 4 ++--
 mx6/libcamera2/OvDevice.cpp   | 4 ++--
 mx6/libcamera2/TVINDevice.cpp | 4 ++--
 mx6/libcamera2/UvcDevice.cpp  | 4 ++--
 6 files changed, 12 insertions(+), 12 deletions(-)

diff --git a/mx6/libcamera2/Ov5640Csi.cpp b/mx6/libcamera2/Ov5640Csi.cpp
index 2db417b..92e48cb 100755
--- a/mx6/libcamera2/Ov5640Csi.cpp
+++ b/mx6/libcamera2/Ov5640Csi.cpp
@@ -130,9 +130,9 @@ status_t Ov5640Csi::initSensorInfo(const CameraInfo& info)
     FLOGI("FrameDuration is %lld, %lld", mMinFrameDuration, mMaxFrameDuration);
 
     i = 0;
-    mTargetFpsRange[i++] = 12;
+    mTargetFpsRange[i++] = 10;
     mTargetFpsRange[i++] = 15;
-    mTargetFpsRange[i++] = 27;
+    mTargetFpsRange[i++] = 25;
     mTargetFpsRange[i++] = 30;
 
     setMaxPictureResolutions();
diff --git a/mx6/libcamera2/Ov5640Mipi.cpp b/mx6/libcamera2/Ov5640Mipi.cpp
index 681605b..85115c9 100755
--- a/mx6/libcamera2/Ov5640Mipi.cpp
+++ b/mx6/libcamera2/Ov5640Mipi.cpp
@@ -124,9 +124,9 @@ status_t Ov5640Mipi::initSensorInfo(const CameraInfo& info)
     FLOGI("FrameDuration is %lld, %lld", mMinFrameDuration, mMaxFrameDuration);
 
     i = 0;
-    mTargetFpsRange[i++] = 12;
+    mTargetFpsRange[i++] = 10;
     mTargetFpsRange[i++] = 15;
-    mTargetFpsRange[i++] = 27;
+    mTargetFpsRange[i++] = 25;
     mTargetFpsRange[i++] = 30;
 
     setMaxPictureResolutions();
diff --git a/mx6/libcamera2/Ov5642Csi.cpp b/mx6/libcamera2/Ov5642Csi.cpp
index cc9c58d..3a02dda 100755
--- a/mx6/libcamera2/Ov5642Csi.cpp
+++ b/mx6/libcamera2/Ov5642Csi.cpp
@@ -123,9 +123,9 @@ status_t Ov5642Csi::initSensorInfo(const CameraInfo& info)
     FLOGI("FrameDuration is %lld, %lld", mMinFrameDuration, mMaxFrameDuration);
 
     i = 0;
-    mTargetFpsRange[i++] = 12;
+    mTargetFpsRange[i++] = 10;
     mTargetFpsRange[i++] = 15;
-    mTargetFpsRange[i++] = 27;
+    mTargetFpsRange[i++] = 25;
     mTargetFpsRange[i++] = 30;
 
     setMaxPictureResolutions();
diff --git a/mx6/libcamera2/OvDevice.cpp b/mx6/libcamera2/OvDevice.cpp
index 1d89b91..8dfb0be 100755
--- a/mx6/libcamera2/OvDevice.cpp
+++ b/mx6/libcamera2/OvDevice.cpp
@@ -162,9 +162,9 @@ status_t OvDevice::initSensorInfo(const CameraInfo& info)
     FLOGI("FrameDuration is %lld, %lld", mMinFrameDuration, mMaxFrameDuration);
 
     i = 0;
-    mTargetFpsRange[i++] = 12;
+    mTargetFpsRange[i++] = 10;
     mTargetFpsRange[i++] = 15;
-    mTargetFpsRange[i++] = 27;
+    mTargetFpsRange[i++] = 25;
     mTargetFpsRange[i++] = 30;
 
     setMaxPictureResolutions();
diff --git a/mx6/libcamera2/TVINDevice.cpp b/mx6/libcamera2/TVINDevice.cpp
index e5562a3..0f6990d 100755
--- a/mx6/libcamera2/TVINDevice.cpp
+++ b/mx6/libcamera2/TVINDevice.cpp
@@ -189,9 +189,9 @@ status_t TVINDevice::initSensorInfo(const CameraInfo& info)
     FLOGI("FrameDuration is %lld, %lld", mMinFrameDuration, mMaxFrameDuration);
 
     i = 0;
-    mTargetFpsRange[i++] = 12;
+    mTargetFpsRange[i++] = 10;
     mTargetFpsRange[i++] = 15;
-    mTargetFpsRange[i++] = 27;
+    mTargetFpsRange[i++] = 25;
     mTargetFpsRange[i++] = 30;
 
     setMaxPictureResolutions();
diff --git a/mx6/libcamera2/UvcDevice.cpp b/mx6/libcamera2/UvcDevice.cpp
index a9bd6d0..8768170 100644
--- a/mx6/libcamera2/UvcDevice.cpp
+++ b/mx6/libcamera2/UvcDevice.cpp
@@ -259,9 +259,9 @@ status_t UvcDevice::initSensorInfo(const CameraInfo& info)
     FLOGI("FrameDuration is %lld, %lld", mMinFrameDuration, mMaxFrameDuration);
 
     i = 0;
-    mTargetFpsRange[i++] = 12;
+    mTargetFpsRange[i++] = 10;
     mTargetFpsRange[i++] = 15;
-    mTargetFpsRange[i++] = 27;
+    mTargetFpsRange[i++] = 25;
     mTargetFpsRange[i++] = 30;
 
     setMaxPictureResolutions();
-- 
1.8.0

